main.py

```diff
@@ -1,10 +1,54 @@
+import json
+
 import init_django_orm  # noqa: F401
 
+from django.utils.timezone import now
+
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        players_data = json.load(file)
+
+    for player_name, player_data in players_data.items():
+        race_data = player_data["race"]
+        race_name = race_data["name"]
+        race_description = race_data.get("description", "")
+        race, _ = Race.objects.get_or_create(
+            name=race_name,
```

Comment:
no need to create extra variables such as `race_name`, `race_description` - since you are not going to reuse them